src/hooks/index.ts

```diff
@@ -2,6 +2,8 @@ export {default as useCombinedRefs} from './useCombinedRefs';
 export {default as useToggleValue} from './useToggleValue';
 export {default as useDidUpdate} from './useDidUpdate';
 export {default as useMeasure} from './useMeasure';
+export {default as useModifiers} from './useModifiers';
+export {default as useThemeProps} from './useThemeProps';
```

Comment:
Do we actually want to export them? Is this for private lib usage?

Reply:
Yes